spec/shr_adverse.txt

```diff
@@ -12,9 +12,11 @@ CodeSystem:		MTH = http://ncimeta.nci.nih.gov
 Element:		AdverseEvent
 Based on:		ClinicalStatement
 Description:	"A record of an untoward or unexpected event in a patient under treatment, and which does not necessarily have to have a causal relationship with that treatment. Adverse events may capture either a workflow breach that may or may not result in harm, but must be documented for quality improvement purposes, or an adverse finding whether or not it can be traceable to an actual workflow breach."
+0..1			Identifier
 				Recorder value is type PatientOrPractitionerOrRelatedPerson
 1..1			Patient
 1..1			OccurrenceTime
+0..1			Type from MedDRAVS
```

Comment:
If our foundation is CTCAE 5.0 for identifying adverse event categories, then we could potentially use NCI Thesaurus codes. Reference the CTCAE 5.0 downloadable spreadsheet from NCI: https://evs.nci.nih.gov/ftp1/CTCAE/CTCAE_5.0/NCIt_CTCAE_5.0.xlsx